modules/test/playwright/tests/frontend-taglib-clay/managementToolbar.spec.ts

```diff
@@ -54,18 +51,140 @@ test.describe('Management Toolbar With Results', () => {
+	test('Assert the New button is displayed as a text button @LPS-144540', async ({
+		page,
+	}) => {
+		await test.step('Check that the new button text is "New"', async () => {
+			await expect(
+				page
+					.locator('#managementToolbarDefaultState')
+					.getByRole('button')
+					.filter({hasText: 'New'})
+			).toBeVisible();
+		});
+	});
+
+	test('Assert the new button is displayed as an icon button in responsive mode @LPS-144540', async ({
+		page,
+	}) => {
+		await test.step('Set the window size to phone size', async () => {
+			await page.setViewportSize({height: 720, width: 360});
+		});
+
+		await test.step('Check the button is an icon', async () => {
+			await expect(
+				page
+					.locator('#managementToolbarDefaultState')
+					.getByRole('button', {name: 'New'})
+					.locator('.lexicon-icon-plus')
+			).toBeVisible();
+		});
+	});
+
+	test('Assert the tooltip message is displayed when hovered over the new button in responsive mode @LPS-144540', async ({
+		page,
+	}) => {
+		await test.step('Set the window size to phone size', async () => {
+			await page.setViewportSize({height: 720, width: 360});
+		});
+
+		await test.step('Hover over the "New" button', async () => {
+			await page
+				.locator('#managementToolbarDefaultState')
+				.getByRole('button', {name: 'New'})
+				.hover();
+		});
+
+		await test.step('Check the tooltip text displays "New"', async () => {
+			await expect(
+				page.locator('.tooltip-inner').getByText('New')
+			).toBeVisible();
+		});
+	});
+
+	test('Assert tooltip messages will be displayed when hovered over the filter and order buttons in responsive mode @LPS-144536', async ({
+		page,
+	}) => {
+		await test.step('Set the window size to phone size', async () => {
+			await page.setViewportSize({height: 720, width: 360});
+		});
+
+		await test.step('Hover over the "Filter" button', async () => {
+			await page
+				.locator('#managementToolbarDefaultState')
+				.getByRole('button', {name: 'Filter'})
+				.hover();
+		});
+
+		await test.step('Check the tooltip text displays "Show Filter Options"', async () => {
+			await expect(
+				page.locator('.tooltip-inner').getByText('Show Filter Options')
+			).toBeVisible();
+		});
+
+		await test.step('Hover over the "Order" button', async () => {
+			await page
+				.locator('#managementToolbarDefaultState')
+				.getByRole('button', {name: 'Order'})
+				.hover();
+		});
+
+		await test.step('Check the tooltip text displays "Show Order Options"', async () => {
+			await expect(
+				page.locator('.tooltip-inner').getByText('Show Order Options')
+			).toBeVisible();
+		});
+	});
+
+	test('Assert the view button displays a double caret icon @LPS-144535', async ({
+		page,
+	}) => {
+		await test.step('Check that the double caret icon is visible', async () => {
+			await expect(
+				page.locator(
+					'#managementToolbarDefaultState .lexicon-icon-caret-double-l'
+				)
+			).toBeVisible();
+		});
+	});
```

Comment:
There are quite a few very simple tests like this one. Could we merge as much tests as we can into one test, so we avoid setup and teardown each time? This can be just one of the steps in the test.